docs/content/querying/unionall.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: doc_page
+---
+# Union All Queries
+Unions multiple query results into single one. All sub queries should be the same type.
```

Comment:
are there any other restrictions also on the sub queries ? 
like - does they need to have same dimensions and metrics ?